vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -265,101 +261,147 @@ external:
+
+  :::note
+  The license secret is already base64-encoded.
+  :::
+
+  ```bash title="Create a Kubernetes secret for license Key"
+  # Set the license key as an environment variable (already base64 encoded)
+  export VCLUSTER_LICENSE_KEY="YOUR_BASE64_ENCODED_LICENSE_KEY_HERE"
+  
+  # Create secret
+  kubectl create -f - <<EOF
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: vcluster-platform-license-key
+    namespace: ${VCLUSTER_NAMESPACE}
+  type: Opaque
+  data:
+    license: ${VCLUSTER_LICENSE_KEY}
+  EOF
+  ```
+  </Step>
+  <Step>
+  **Optional**: Create a secret to authenticate to your private registry.
+
+  If your private registry requires authentication, create a Kubernetes secret that stores your login credentials.
+  Place this secret in the same namespace where the vCluster is deployed so the control plane can pull the required container images.
+
+  ```bash title="Create Kubernetes secret for login credentials"
+  kubectl create -f - <<EOF
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: registry-credentials-secret-name
+    namespace: ${VCLUSTER_NAMESPACE}
+  type: Opaque
+  data:
+    # Credentials based on your private registry
+  EOF
+  ```
 
-```yaml title="vcluster.yaml configuration"
-controlPlane:
-  advanced:
-    defaultImageRegistry: my-private-registry:5000/vcluster/
-    serviceAccount:
-      imagePullSecrets:
-        - name-of-your-image-pull-secret
-  backingStore:
-    etcd:
-      embedded:
-        enabled: true
-  coredns:
-    embedded: true
+  </Step>
 
-external:
-  platform:
-    apiKey:
-      secretName: vcluster-platform-api-key
-```
+  <Step>
+  Review your `vcluster.yaml` file.
+
+  Use the `vcluster.yaml` file with the necessary configuration for your air-gapped vCluster deployment: 
+
+  ```yaml title="vcluster.yaml configuration"
+  controlPlane:
+    advanced:
+      defaultImageRegistry: ecr.io/myteam # Replace with your private image registry
+      serviceAccount:
+        imagePullSecrets: 
+          - name: registry-credentials-secret-name # Replace with the name of the registry credentials secret
+      workloadServiceAccount: 
+        imagePullSecrets:
+          - name: registry-credentials-secret-name # Replace with the name of the registry credentials secret
+    
+  external:
+    platform:
+      apiKey:
+        secretName: vcluster-platform-license-key
+  ```
 
   </Step>
+  <Step>
+  Pull the vCluster Helm chart from the private registry.
+  
+  ```bash title="Pull the Helm chart"
+  export VCLUSTER_VERSION=0.25.0 # Replace with the chart version that was pushed into your private registry
+  export REGISTRY=myecr.io/team # Replace with the private registry
+  
+  ## Log in to the Helm registry if you haven't
+  helm registry login ${REGISTRY} \
+    --username myuser \
+    --password mypassword
+  
+  helm pull oci://${REGISTRY}/vcluster:v${VCLUSTER_VERSION}
```

Comment:
There shouldn't be a `v`: `helm pull oci://${REGISTRY}/vcluster:${VCLUSTER_VERSION}`

Reply:
I also added `/charts` after REGISTRY